ivy/array/linear_algebra.py

```diff
@@ -90,37 +90,44 @@ def cross(
         axis: int = -1,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
-        """
-        ivy.Array instance method variant of ivy.cross. This method simply wraps the
-        function, and so the docstring for ivy.cross also applies to this method
-        with minimal changes.
+        """ivy.Array instance method variant of ivy.cross. This method simply wraps the
+            function, and so the docstring for ivy.cross also applies to this method
+            with minimal changes.
 
         Parameters
         ----------
         self
-            first input array. Should have a numeric data type.
+            first input array that is calling the function. Should have a numeric data type. 
         x2
-            second input array. Must be compatible with ``self``
-            (see :ref:`broadcasting`). Should have a numeric data type.
+            second input array. Must have the same shape as x1. Should have a numeric data
+            type.
         axis
-            the axis (dimension) of x1 and x2 containing the vectors for which to
-            compute (default: -1) the cross product.vIf set to -1, the function
-            computes the cross product for vectors defined by the last axis (dimension).
-            Default: -1.
+            the axis (dimension) of x1 and x2 containing the vectors for which to compute
+            the cross product.vIf set to -1, the function computes the cross product for
+            vectors defined by the last axis (dimension). Default: -1.
         out
-            optional output array, for writing the result to. It must have a shape that
-            the inputs broadcast to.
+            optional output array, for writing the result to. It must have a shape that the
+            inputs broadcast to.
 
         Returns
         -------
         ret
-            an array containing the element-wise products. The returned array must
-            have a data type determined by :ref:`type-promotion`.
+            an array containing the cross products. The returned array must have a data
+            type determined by Type Promotion Rules.
 
-        Examples
+        This function conforms to the `Array API Standard
```

Comment:
this does not need to be added for array instance methods, check [here](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html).